extensions/blocks/podcast-player/style.scss

```diff
@@ -218,6 +233,11 @@ $player-float-background: $light-gray-200;
 		padding: 0 $track-v-padding;
 	}
 
+	&.is-default .jetpack-podcast-player__track-title {
```

Comment:
I wonder if grouping these `is-default` changes together would be easier to read.

Reply:
Done!